cocos/audio/apple/AudioCache.mm

```diff
@@ -255,7 +252,10 @@ static ALvoid  alBufferDataStaticProc(const ALint bid, ALenum format, ALvoid* da
                 remainingFrames -= framesRead;
             }
 
-            CCASSERT(_framesRead == originalTotalFrames, "The read frame count isn't equal to orignal frame count!");
+            if (_framesRead < originalTotalFrames)
```

Comment:
@minggo , it may trigger this assertion while playing very small audio file.
As the title said, 'ExtAudioFileRead' may return 0 frame while it doesn’t reach the end of very short audio file.
This is a bug of Apple SDK. :(